### PR TITLE
[1LP][RFR] Fixed case sensitive check.

### PR DESCRIPTION
**Pull request by hhovsepy**
{{pytest: cfme/tests/middleware/test_middleware_domain_server.py -v --use-provider hawkular}}

### CODE REVIEW

**Comment by quarckster** [comment]
Looks good, but PRT failed.

File: cfme/tests/middleware/test_middleware_domain_server.py [hunk 1]
```diff
@@ -97,7 +97,8 @@ def test_domain_server_stop_start(provider, domain_server):
     """
     verify_server_running(provider, domain_server)
     domain_server.stop_server()
-    verify_server_stopping(provider, domain_server)
+    # potential fail if it stops fast
+    # verify_server_stopping(provider, domain_server)
```
Comment (mshriver) [inline cfme/tests/middleware/test_middleware_domain_server.py]: **Required**

use soft_assert, or at the very minimum catch the assertionerror and log a warning.

**Comment by hhovsepy** [comment]
Please ignore the failures, BZ is created for them https://bugzilla.redhat.com/show_bug.cgi?id=1520324

**Comment by hhovsepy** [comment]
@mshriver have fixed the soft assert stuff.

**Pull request merged**